Formula/llvm@11.rb

```diff
@@ -146,6 +150,7 @@ def install
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
       args << "-DLLVM_CREATE_XCODE_TOOLCHAIN=#{MacOS::Xcode.installed? ? "ON" : "OFF"}"
+      args << "-DRUNTIMES_CMAKE_ARGS=-DCMAKE_INSTALL_RPATH=#{rpath}"
```

Comment:
```suggestion
```
This isn't needed. The change in `LC_RPATH` entries for runtimes changed (i.e. regressed) in LLVM 12.